lib/auth/recordingmetadata/recordingmetadatav1/recordingmetadata.go

```diff
@@ -233,6 +228,11 @@ loop:
 				return nil
 
 			case *apievents.Resize:
+				// TODO(zmb3): consider not handling this event at all, and instead
```

Comment:
I agree with you here. Lets pass the raw event to the generator and it's responsible for handling the correct resize for its type